app/assets/stylesheets/sufia/_form.scss

```diff
@@ -23,3 +23,10 @@ td.toggle {
     &:not(.active) { @extend .btn-default; }
   }
 }
+
+/* for profile page highlighted works */
+tr[id^='trophyrow_'] {
```

Comment:
I suspect this might be very slow. The browser will have to run a regex on every id of every tr on the page.  Can you add a class to the row instead?